app/scripts/controllers/membership.js

```diff
@@ -364,5 +363,25 @@ angular
                 }
               });
             });
+
+
+          DataService
+            .list('serviceaccounts', requestContext)
+            .then(function(resp) {
+              var serviceAccounts = _.map(resp.by('metadata.name'), function(sa) {
+                return sa.metadata.name;
+              });
```

Comment:
Doesn't matter to me, the update will put the 2nd role on the SA either way.  Prefer to eliminate it from the dropdown?